hermes-consumers/src/main/java/pl/allegro/tech/hermes/consumers/consumer/receiver/kafka/HermesMessageContentReader.java

```diff
@@ -0,0 +1,34 @@
+package pl.allegro.tech.hermes.consumers.consumer.receiver.kafka;
+
+import org.apache.kafka.clients.consumer.ConsumerRecord;
+import pl.allegro.tech.hermes.api.ContentType;
+import pl.allegro.tech.hermes.api.Topic;
+import pl.allegro.tech.hermes.common.message.wrapper.MessageContentWrapper;
+import pl.allegro.tech.hermes.common.message.wrapper.UnsupportedContentTypeException;
+import pl.allegro.tech.hermes.common.message.wrapper.UnwrappedMessageContent;
+
+public class HermesMessageContentReader implements MessageContentReader {
```

Comment:
I'm not sure `Hermes*` is an appropriate name for an implementation of an interface. :) In my opinion it's better to call it Simple*, Basic* ... :]

Reply:
Naming is probably the most difficult thing in a software engineering... :D

Reply:
Ok, I renamed it to `BasicMessageContentReader`.